crossplane-provider-keycloak.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: crossplane-provider-keycloak
+  version: 1.7.0
+  epoch: 0
+  description: Official Keycloak Provider for Crossplane by Upbound
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - terraform
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - curl
+      - go
+      - gzip
+      - terraform
+      - up
+  environment:
+    CGO_ENABLED: 0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 12dcc112937e0b4451cd1080dd5217ebfacc2b4b
+      repository: https://github.com/crossplane-contrib/provider-keycloak
+      tag: v${{package.version}}
+      recurse-submodules: true
+
+  - uses: go/bump
+    with:
+      deps: github.com/cloudflare/circl@v1.3.7 github.com/hashicorp/go-retryablehttp@v0.7.7
+
+  - name: Code Generation
+    runs: |
+      export GOBIN="$GOPATH/bin"
+      export PATH=$PATH:$GOPATH/bin
+      go install golang.org/x/tools/cmd/goimports@latest
+      go run cmd/generator/main.go $(pwd)
+      make generate
+
+  - uses: go/build
+    with:
+      output: crossplane-keycloak-provider
+      packages: ./cmd/provider
+
```

Comment:
Should we also pass `LDFLAGS` for _version_?

Also it seems they build the `generator` package, do we also need it?

Reply:
i was looking the image and there is no generator, i think that is only used to generate the code gen